pyproject.toml

```diff
@@ -77,6 +80,7 @@ dev = [
     "mapbox-vector-tile",
     "jinja2",
     "tqdm",
+    "trio",
```

Comment:
This would've been better as `anyio[trio]` to guarantee a trio version compatible with AnyIO.